sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -45,47 +45,99 @@ func init() {
 }
 
 func main() {
-	os.Exit(_main())
+	if err := _main(); err != nil {
+		log.L.Fatalf("%v", err)
+	}
 }
 
-func _main() int {
-	// Parse command-line arguments
+func _main() error {
```

Comment:
Instead of returning the result of having called the command, the `cli.App` type should be returned for direct use in testing and have that `cli.App.Run` called from the real `main`. This gives a little more control to testing entrypoints where `Run` can optionally be called.

If that were changed, I'd rename this to `func App() cli.App` (`App` because it stands out a bit more than `app`, not because it needs to exported and because I personally like it :P)